apis/kueue/v1alpha1/multikueue_types.go

```diff
@@ -53,6 +53,16 @@ type KubeConfig struct {
 type MultiKueueClusterSpec struct {
 	// Information how to connect to the cluster.
 	KubeConfig KubeConfig `json:"kubeConfig"`
+
+	// A label value used to track the creator of workloads in the worker cluster.
+	// This is used by the garbage collector to identify remote objects that ware created via
+	// this cluster and delete them if their local counterpart no longer exists.
+	//
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
+	// +kubebuilder:validation:MaxLength=63
+	// +kubebuilder:default=multikueue
+	Origin string `json:"origin,omitempty"`
```

Comment:
I believe however  this should be configurable at the global Kueue level. As discussed here https://github.com/kubernetes-sigs/kueue/pull/1643/files#r1472664011 - the only purpose of making this configurable is to separate workloads and jobs if multiple MultiKueue instances share the same worker. This can be achieved at the global MultiKueue configuration level. No need to set it per cluster.

Reply:
global sounds more appropriate

Reply:
done